examples/command/portals/kafka/instaclustr/docker/run.sh

```diff
@@ -0,0 +1,116 @@
+    #
+    # The enroll command then asks this project’s membership authority to sign and issue
+    # a credential that attests that your identifier is a member of this project. Since your
+    # account in Orchestrator is the creator and hence first administrator on this new project,
+    # the membership authority issues this credential. The enroll command stores the
+    # credential for later use and exits.
+    ockam enroll
+
+    # Create an enrollment ticket to enroll the identity used by an ockam node that will run
+    # adjacent to the Instaclustr server in instaclustr_operator's network.
+    #
+    # The identity that enrolls with the generated ticket will be given a cryptographically
+    # attestest project membership credential issue by the membership authority.
+    #
+    # The identity will also allowed to create a relay in the project at the address `instaclustr`.
+    instaclustr_operator_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay instaclustr)
+
+    # Create an enrollment ticket to enroll the identity used by an ockam node that will run
+    # adjacent to the Instaclustr client app in application_team's network.
+    #
+    # The identity that enrolls with the generated ticket will be given a cryptographically
+    # attestest project membership credential issue by the membership authority.
+    application_team_consumer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+    application_team_producer_ticket=$(ockam project ticket --usage-count 1 --expires-in 10m --relay '*')
+
+    # Invoke `docker-compose up` in the directory that has instaclustr_operator's configuration.
+    # Pass the above enrollment ticket as an environment variable.
+    #
+    # Read instaclustr_operator/docker-compose.yml to understand the parts that are provisioned
+    # in instaclustr_operator's virtual private network.
+    echo; pushd instaclustr_operator; ENROLLMENT_TICKET="$instaclustr_operator_ticket" BOOTSTRAPSERVER="$bootstrap_server" docker compose up -d; popd
+
+    # Invoke `docker-compose up` in the directory that has application_team's configuration.
+    # Pass the above enrollment ticket as an environment variable.
+    #
+    # Read application_team/docker-compose.yml to understand the parts that are provisioned
+    # in application_team's virtual private network.
+    echo; pushd application_team; PRODUCER_ENROLLMENT_TICKET="$application_team_producer_ticket" CONSUMER_ENROLLMENT_TICKET="$application_team_consumer_ticket" docker compose up; popd
+}
+
+# Cleanup after the example - `./run.sh cleanup`
+# Remove all containers and images pulled or created by docker compose.
+cleanup() {
+    instaclustr_auth
+    ./cluster_manager.sh cleanup
+    pushd instaclustr_operator; docker compose down --rmi all --remove-orphans; popd
+    pushd application_team; docker compose down --rmi all --remove-orphans; popd
+}
+
+instaclustr_auth() {
```

Comment:
It would be nice to skip this function if the user already has the `INSTACLUSTR_USER_NAME` and `INSTACLUSTR_API_KEY` vars.